test_data.rb

```diff
@@ -240,11 +240,13 @@ def test_localized_info_different_than_name
   end
 
   def test_suggestable_has_carrier
+    stations_to_ignore = %w(68192)
```

Comment:
:thinking: Have you considered adding a meta-station, something like "Madrid Cercanias zone", then make all cercanias children of it, and eventually make "Madrid Cercanias zone" children of Madrid?

Reply:
>  Have you considered adding a meta-station, something like "Madrid Cercanias zone", then make all cercanias children of it, and eventually make "Madrid Cercanias zone" children of Madrid?

Perhaps it is too complicated - what would be the advantages of doing this?

Reply:
I was thinking that it might avoid test changes, if the parent station gets some Renfe ID on it.